Makefile

```diff
@@ -366,9 +366,9 @@ venv:
 setup: venv
 	$(MAKE) -C tools
 	$(PYTHON) tools/decompress_baserom.py $(VERSION)
-# TODO: for now, we only extract ROM segments and assets from the Debug ROM
+	$(PYTHON) tools/extract_baserom.py $(BASEROM_DIR)/baserom-decompressed.z64 -o $(BASEROM_SEGMENTS_DIR) --dma-start `cat $(BASEROM_DIR)/dma_start.txt` --dma-names $(BASEROM_DIR)/dma_names.txt
```

Comment:
Curious as to the reasoning to pass everything by arguments, instead of just passing a version like `decompress_baserom.py`?

Reply:
I can live with the inconsistency but if I had to choose I'd like "file paths in the makefile" better than "hardcoded in a script"

Reply:
Using arguments makes it easier to run the tool outside of the build system, either for testing or because it's already useful on its own. For example, I've already used `extract_baserom.py` to extract segments from the non-matching built ROM to make it easier to study. It could be nice if `decompress_baserom.py` took arguments too, but it does a lot more stuff (e.g. checking hashes), and I can z64decompress instead of running it on its own, so I can live with that too.